### ISSUE TITLE
Mutlilayer Perceptron - Base

### ISSUE THREAD

**Issue by ashwani-rathee** [Algo-Phantoms/Algo-ScriptML#46]
Task: Set up the basic Mutlilayer Perceptron code functionality that works
Level-3 : 25 Points

**Comment by JanhaviSoni** [comment]
Assign this to me

**Comment by ashwani-rathee** [comment]
@JanhaviSoni  You have been assigned the task to setup the base for Multilayer Perceptron.
Try to submit a PR in 5 days.

**Comment by JanhaviSoni** [comment]
@ashwani-rathee I wanted to ask that can we use modules or everything from scratch . Also we just need to write how the algo works or implement it using data set

**Comment by ashwani-rathee** [comment]
@JanhaviSoni You can use numpy,and that should be enough for algorithm part,you have to just kind of prove that your method works(so yeah you would need a simple example).

**Comment by ashwani-rathee** [comment]
@JanhaviSoni  What is your progress on this?Any issues?How can I help?
2days to a PR

**Comment by JanhaviSoni** [comment]
@ashwani-rathee It will take me more time to work on implementation of Mulilayer Perceptron. Till that time can I be assigned Multiple Linear Regression and you could unassign Multilayer Perceptron.

**Comment by jprakash-1** [comment]
@ashwani-rathee I am Gssoc participant can u assign me Multilayer Perceptron.

**Comment by ashwani-rathee** [comment]
@jprakash-1 would you like to get this assigned?

**Comment by jprakash-1** [comment]
Yes, @ashwani-rathee  I would like to work on this. Could you please tell me how many hidden layers do I have to implement? And any other relevant detail I need, to implement this? 

Thank you

**Comment by Sibasish-Padhy** [comment]
@ashwani-rathee I would like to contribute to this issue cn you please assign me this issue under gssoc label

**Comment by saumyasinghz** [comment]
@ashwani-rathee Please assign this issue to me.

**Comment by ashwani-rathee** [comment]
This needs to be assigned

**Comment by ashwani-rathee** [comment]
@saumyasinghz Sure, go ahead

**Comment by subhangi2731** [comment]
@ashwani-rathee please assign me

**Comment by geekquad** [comment]
Go ahead @subhangi2731.
